lib/render/site-award.php

```diff
@@ -128,6 +136,20 @@ function RenderAwardGroup($awards, $title): void
     echo "</div>";
 }
 
+function RenderCounter($icon, $text, $numItems, $numHidden): string
+{
+    $tooltip = $icon . "  " . $numItems . " " . $text;
```

Comment:
Personally, I don't think the icon belongs in the tooltip. It's redundant with the icon that the user hovered over to get the tooltip.